.bldr.toml

```diff
@@ -1552,6 +1552,7 @@ paths = [
   "api/external/common/*",
   "api/interservice/authz/*",
   "api/interservice/teams/*",
+  "components/authz-service/*",
```

Comment:
This is a bit unfortunate. I wonder if it's either code we could put into `lib/` or proto files we could put into `api/`. 🤔

Reply:
I think this is getting picked up because we are using projectdiff actually. I'll move that into lib.